hat/assets/js/apps/Iaso/components/Buttons/EditIconButton.tsx

```diff
@@ -13,17 +13,20 @@ type Props = {
     onClick: () => void;
     message?: IntlMessage;
     disabled?: boolean;
+    overridedIcon?: any;
 };
 
 export const EditIconButton: FunctionComponent<Props> = ({
     onClick,
     message,
     disabled = false,
+    overridedIcon,
```

Comment:
overridedIcon --> overrideIcon ( to stay consistent)
```suggestion
    overrideIcon,
```